eos/parameters/hadron-properties.yaml

```diff
@@ -446,161 +453,162 @@ groups:
           central: +0.204
           min:     +0.197
           max:     +0.211
+          latex:    '$f^{K^*}_{\parallel}(1\,\GeV)$'
           comments: 'cf. [BSZ2015], table 1'
       'K^*::a_1_perp@1GeV' :
           central: +0.04
           min:     +0.01
           max:     +0.07
+          latex:    '$a^{K^*}_{1,\perp}(1\,\GeV)$'
           comments: 'cf. [BSZ2015], table 1'
       'K^*::a_2_perp@1GeV' :
           central: +0.10
           min:     +0.02
           max:     +0.18
+          latex:    '$a^{K^*}_{2,\perp}(1\,\GeV)$'
           comments: 'cf. [BSZ2015], table 1'
       'K^*::f_perp@1GeV' :
           central: +0.159
           min:     +0.153
           max:     +0.165
+          latex:    '$f^{K^*}_{\perp}(1\,\GeV)$'
           comments: 'cf. [BSZ2015], table 1'
       # B->K LCDA Parameter
-      'B->K::a_1@1GeV' :
+      'K::a_1@1GeV' :
           central: +0.06
           min:     +0.03
           max:     +0.09
+          latex:    '$a^{K}_{1}(1\,\GeV)$'
           comments: ' // cf. [BBL2006], Table 3'
-      'B->K::a_2@1GeV' :
+      'K::a_2@1GeV' :
           central: +0.25
           min:     +0.10
           max:     +0.4
+          latex:    '$a^{K}_{2}(1\,\GeV)$'
           comments: ' // cf. [BBL2006], Table 3'
-      'B->K::a_4@1GeV' :
+      'K::a_4@1GeV' :
           central: -0.015
           min:     -0.115
           max:     +0.085
+          latex:    '$a^{K}_{4}(1\,\GeV)$'
           comments: ' // cf. [BZ2004v3], Eq. (24)'
-      'B->K::a_1@2.2GeV' :
+      'K::a_1@2.2GeV' :
           central: +0.048
           min:     +0.024
           max:     +0.071
+          latex:    '$a^{K}_{1}(2.2\,\GeV)$'
           comments: ' // cf. [BBL2006], Table 3 and cf. [BHP2007] App. A, pp. 24-25'
-      'B->K::a_2@2.2GeV' :
+      'K::a_2@2.2GeV' :
           central: +0.174
           min:     +0.070
           max:     +0.278
+          latex:    '$a^{K}_{2}(2.2\,\GeV)$'
           comments: ' // cf. [BBL2006], Table 3 and cf. [BHP2007] App. A, pp. 24-25'
-      'B->K::a_4@2.2GeV' :
+      'K::a_4@2.2GeV' :
           central: -0.0089
           min:     -0.0679
           max:     +0.0502
+          latex:    '$a^{K}_{4}(2.2\,\GeV)$'
           comments: ' // cf. [BZ2004v3], Eq. (24) and cf. [BHP2007] App. A, pp. 24-25'
       # B->K^*, K^* LCDA parameters
-      'B->K^*::a_1_par' :
-          central: +0.1
-          min:     +0.03
-          max:     +0.17
-          comments: ''
-      'B->K^*::a_2_par' :
-          central: +0.1
-          min:     +0.0
-          max:     +0.2
-          comments: ''
-      'B->K^*::a_1_perp' :
-          central: +0.1
-          min:     +0.03
-          max:     +0.17
-          comments: ''
-      'B->K^*::a_2_perp' :
-          central: +0.1
-          min:     +0.0
-          max:     +0.2
-          comments: ''
       'B->K^*::f_Kstar_par' :
           central: +0.217
           min:     +0.212
           max:     +0.222
+          latex:    '$f^{K^*}_{\parallel}$'
```

Comment:
Don't we want to have `B` somewhere in the latex representation? e.g. `$f^{B\to K^*}_{\parallel}$`

Reply:
Usually I sticked to the syntax: parametername^{prefix, suffix}, depending on context I did not always stick to this. Here I decided to just use K^* because one of the tasks was including to retire some of the "B->K^*" parameters.

Reply:
I think this point we need to discuss, really, there are mixed parameters B->K^*:: ... and K^*:: ... and I doubt we want those mixed prefixes.
The "B->", also is mostly redundant in LCDA parameters - I think.

Reply:
Ok fine. I agree that the full specification is not always needed. The names just need to be consistent and self explanatory I think.

Reply:
Ok, this is an awkward corner case. This is the K^* tensor decay constant, as used in the B->K^*ll QCDF code. We can leave this one as is and clean it up (i.e., remove it) later....